src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,26 @@
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={classNames('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+    >
```

Comment:
please, format your params properly, will reject next time
```suggestion
    <article 
      className={classNames(
        'TodoInfo',
        { 'TodoInfo--completed': completed }
      )}
    >
```